modules/nw-egress-ips-about.adoc

```diff
@@ -73,29 +73,60 @@ When a pod matches the selector for multiple `EgressIP` objects, there is no gua
 [id="nw-egress-ips-node-architecture_{context}"]
 == Architectural diagram of an egress IP address configuration
 
-The following diagram depicts an egress IP address configuration. The diagram describes the relationship between three nodes in a cluster and four pods running on that cluster in two different namespaces.
+The following diagram depicts an egress IP address configuration. The diagram describes four pods in two different namespaces running on three nodes in a cluster. The nodes are assigned IP addresses from the `192.168.126.0/18` CIDR block on the host network.
 
 // Source: https://github.com/redhataccess/documentation-svg-assets/blob/master/for-web/121_OpenShift/121_OpenShift_engress_IP_Topology_1020.svg
 image::nw-egress-ips-diagram.svg[Architectural diagram for the egress IP feature.]
 
 Both Node 1 and Node 3 are labeled with `k8s.ovn.org/egress-assignable: ""` and thus available for the assignment of egress IP addresses.
 
-The following `EgressIP` object describes a configuration that selects all pods in the `namespace1` namespace, with the `192.168.126.10` and `192.168.126.102` egress IP addresses specified.
+The dashed lines in the diagram depict the traffic flow from `pod1`, `pod2`, and `pod3` traveling through the pod network to egress the cluster from Node 1 and Node 3. When an external service receives traffic from any of these pods, the source IP address is either `192.168.126.10` or `192.168.126.102`.
```

Comment:
You have `pod1` etc in backticks, but not "Node 1" etc. Is there a reason these are different, or should they both be styled the same?

Reply:
@bergerhoffer, I'm not sure why exactly I did it that way. It's not consistent with the digram, and it probably should be. I'll change it.